src/single-spa.js

```diff
@@ -93,6 +93,7 @@ export function reset() {
 		return result;
 	}
 
+	const originalReplaceState = window.history.replaceState;
```

Comment:
This was just a bad oversight on my part not to have this here. See above where I create a `originalPushState` variable, but then I never did so for `originalReplaceState`